swaggerOptions.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module '*.json' {
```

Comment:
If you want to declare or expand some types, create a @types folder in src and declare it there.

Reply:
Done